verify.cmd

```diff
@@ -0,0 +1,49 @@
+@echo off
+
+echo .
+echo Android SDK's required: 25, 26 and 27
+pushd %ProgramFiles(x86)%\Android\android-sdk\platforms
+echo . 
+echo Available SDK's:
+echo %ProgramFiles(x86)%\Android\android-sdk\platforms
+echo .
+echo Open SDK Manager:
+echo  %ProgramFiles(x86)%\Android\android-sdk\SdkManager.exe
+echo .
+echo Validating available SDK's ...
+if not exist android-25 echo Please install Android SDK v25 (Open SDK Manager from above location)
+if not exist android-26 echo Please install Android SDK v26 (Open SDK Manager from above location)
+if not exist android-27 echo Please install Android SDK v27 (Open SDK Manager from above location)
```

Comment:
cool...can we install automatically?

Reply:
Next step will be to kick off the script which will install. For now I'm building out the awareness (and I just added the link to where to install from)

Reply:
awesome